specificatie/openapi.yaml

```diff
@@ -693,8 +694,23 @@ components:
         maxItems: 6
         items:
           type: number
+        example: [196733.510, 439931.890, 196833.510, 440031.890]
       explode: false
 
+    pandIdentificaties:
+      description: "Een lijst met identificaties van panden uit de BAG. Deze is 16 cijfers lang."
+      name: pandIdentificaties
+      in: query
+      required: false
+      schema:
+        type: array
+        items:
+          type: string
+          pattern: '^[0-9]{16}$'
+          minItems: 1
```

Comment:
Vraagje, wat voegt deze _minItems: 1_ hier toe? 
Bij het parameter component _bbox_ hierboven wordt er binnen _items:_ geen _minitems_ aangebracht is er een reden voor dat deze daar niet wordt toegevoegd en hier wel of is het gewoon inconsistentie?

Reply:
Dit minitems staat verkeerd, die moet 2 spaties naar links want geldt voor array en niet voor string in een array. Bij bbox staat minItems: 4 bij array.